subjects/git/README.md

```diff
@@ -0,0 +1,323 @@
+## Git Ready
+
+### Introduction
+
+This project is designed to introduce you to the world of version control and collaboration using **Git**. Git is a powerful and widely used tool for tracking changes in your projects, collaborating with others, and ensuring the integrity of your code.
+
+Throughout this project, you will embark on a journey of progressively building your Git skills. Starting from the basics, you'll gradually explore more advanced topics, equipping yourself with the essential knowledge and practices for effective version control and collaboration.
+
+Let's Git ready for it!
+
+### Instructions
+
+To begin, create a `work` directory and organize all your tasks within it. Each exercise should be encapsulated in its own file, named after the corresponding task for clarity and ease of reference.
```

Comment:
I am not sure the requirements to have one file per section/task makes sense looking at the audit. I would remove the sentence `Each ...` or how are the "tasks files" supposed to be created/structured?

Reply:
this is meant to say that they have to create the structure like this:
```
Work/
│
├── Name_Exercise_1.md
├── Name_Exercise_2.md
├── Name_Exercise_3.md
├── Name_Exercise_4.md
└── Documentation/
    ├── Solutions.md
```
    
 the actual solution with the commands will be a file that they have to show the auditor during the review, this way of putting the actual work they did will make it easier for the auditor to go check the git history of each exercise if needed